### ISSUE TITLE
Print-out of nway-calibrate-cutoff.py

### ISSUE THREAD

**Issue by sfzastro** [JohannesBuchner/nway#19]
Greetings,

Should `efficiency[i]` in the last line of `nway-calibrate-cutoff.py` be `efficiency[i] * 100`? The print-out of nway-calibrate-cutoff.py seems strange to me.

```bash
For a false detection rate of <1%
--> use only counterparts with p_any>1.00 (0.00% of matches)

For a false detection rate of <3%
--> use only counterparts with p_any>0.97 (0.47% of matches)

For a false detection rate of <5%
--> use only counterparts with p_any>0.89 (0.59% of matches)

For a false detection rate of <10%
--> use only counterparts with p_any>0.60 (0.74% of matches)
```

**Comment by JohannesBuchner** [comment]
Yes, it should be!

**Comment by JohannesBuchner** [comment]
If you have time to make a pull request, that would be greatly appreciated

**Comment by JohannesBuchner** [comment]
fixed by #21
